docker-compose.yml

```diff
@@ -1,49 +1,87 @@
 version: '2'
+
 services:
 
-  postgres:
-    image: postgres
+  database:
+    image: camptocamp/postgis:9.5
```

Comment:
@Vampouille can we rely on official image here?

Reply:
There is no official postgis image :
https://store.docker.com/search?q=postgis&source=verified&type=image
and we need postgis extension for geoserver database.

Reply:
Ok then I'll go to create a geonode official through docker hub by forking your repo